app/scripts/filters/util.js

```diff
@@ -418,6 +418,11 @@ angular.module('openshiftConsole')
       return url;
     };
   })
+  .filter('editYamlURL', function(Navigate) {
```

Comment:
we are getting enough of these i'm starting to think they need their own filters/navigate.js, not necessarily in this PR though